### PR TITLE
ovn-e2e tests on the on the cluster network operator using the step registry

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
trying out a step registry job on a repo that does not have any yet. Not sure if any additional setup is required

### CODE REVIEW

**Comment by dcbw** [comment]
@JacobTanenbaum does this replace https://github.com/openshift/release/pull/7651 ?

**Comment by JacobTanenbaum** [comment]
@dcbw yes

**Comment by JacobTanenbaum** [comment]
@stevekuznetsov is there something keeping this from merging? The step registry is doing everything correct it seems like the e2e test itself has failed

File: ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.5.yaml [hunk 1]
```diff
@@ -56,3 +56,8 @@ tests:
   openshift_installer:
     cluster_profile: gcp
     upgrade: true
+- as: e2e-ovn-steps
```
Comment (dcbw) [inline ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.5.yaml]: @JacobTanenbaum should this go into ./ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml (eg master) instead?

File: ci-operator/step-registry/origin/e2e-ovn/origin-e2e-ovn-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: origin-e2e-ovn
```
Comment (dcbw) [inline ci-operator/step-registry/origin/e2e-ovn/origin-e2e-ovn-workflow.yaml]: @JacobTanenbaum origin-e2e-aws-ovn?

File: ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.5.yaml (hunk 1 above)
Comment (JacobTanenbaum) [reply ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.5.yaml]: done

File: ci-operator/step-registry/origin/e2e-ovn/origin-e2e-ovn-workflow.yaml (hunk 2 above)
Comment (JacobTanenbaum) [reply ci-operator/step-registry/origin/e2e-ovn/origin-e2e-ovn-workflow.yaml]: done

**Comment by stevekuznetsov** [comment]
/approve
/hold

@bbguimaraes feel free to cancel the hold when you look

**Pull request merged**